src/main/scala/nl.knaw.dans.easy.sword2/DepositHandler.scala

```diff
@@ -289,9 +289,9 @@ object DepositHandler {
     }
 
     def extract(file: File, outputPath: String): Unit = {
-      new ZipFile(file.getPath) {
-        setFileNameCharset(StandardCharsets.UTF_8.name)
-      }.extractAll(outputPath)
+      import better.files._
```

Comment:
you should know better!!! 😉
Imports go on top of the file...

Reply:
Well, in this case I think that is not such a good idea. It will cause a lot of changes because we would need to import `java.io.File` under an alias.

Reply:
👍never thought about that! Good point.